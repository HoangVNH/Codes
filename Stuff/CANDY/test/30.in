10
27
VVVVXXXXXXXXXXXXXXXXDXXXXXV
35
VVVVDDDXXVVDDDDXXVVVVXXXXXVVVVDDDDD
30
VVVDDDDDDDDDDDDVDDXXXXXXXXVVDD
42
XXXDDXDDDDDXDDDDDVVVVDDVVVDDDXXXXXVVDDDDVV
34
DDXXXDDDDDDDDVVVDDDDDVXXDDVVVXXDDX
48
XXXXXXXXXXXXXXVVXDDDDDDXXXVVVVVVXXXXVXXXXXXXXXXV
24
DDDDXXXXXXXXXXXDDDVVVVVD
40
XVVVVVVVDVVVVVVVVVVVVVDDDVXXVXXXXXXDDXXV
20
DDXXXVXXXXXDDDDXXXXX
25
DDDXXVVVVVVDDDDDXXXXXXXXX
