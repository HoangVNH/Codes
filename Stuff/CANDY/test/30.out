25
34
28
41
33
46
23
39
19
24
